CrowdStrike/glide-core
# Dropdown's first option's tooltip is shown when opened via click

## 🚀 Description

- Dropdown no longer shows a tooltip on the first option when it has overflowing text and Dropdown is opened via click.
- Tag no longer has a `textContent` getter.
- Minor Storybook styling tweaks.

## 📋 Checklist



- I have followed the [Contributing Guidelines](https://github.com/crowdstrike/glide-core/blob/main/CONTRIBUTING.md).
- I have added tests to cover new or updated functionality.
- I have added or updated Storybook stories.
- I have [localized](https://github.com/CrowdStrike/glide-core/blob/main/CONTRIBUTING.md#translations-and-static-strings) new strings.
- I have followed the [ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/patterns/) or met with the Accessibility Team.
- I have included a [changeset](https://github.com/CrowdStrike/glide-core/blob/main/CONTRIBUTING.md#versioning-a-package).
- I have scheduled a design review.

## 🔬 How to Test

1. Navigate to Dropdown in Storybook.
2. Open Dropdown via click.
3. Verify the first option's tooltip isn't shown.
4. Hover the first option.
5. Verify the first option's tooltip is shown.
6. Close Dropdown.
7. Open Dropdown via Space.
8. Verify the first option's tooltip is shown.
9. Arrow down then back up to the first option.
10. Verify the first option's tooltip is shown.

## 📸 Images/Videos of Functionality

N/A
